Sources/FaunaDB/Value.swift

```diff
@@ -242,6 +278,29 @@ extension RefV: Equatable {
     }
 }
 
+public struct Native {
+    public static let CLASSES: RefV = RefV("classes")
+    public static let INDEXES: RefV = RefV("indexes")
+    public static let DATABASES: RefV = RefV("databases")
+    public static let KEYS: RefV = RefV("keys")
+    public static let FUNCTIONS: RefV = RefV("functions")
+    public static let TOKENS: RefV = RefV("tokens")
+    public static let CREDENTIALS: RefV = RefV("credentials")
+
+    public static func fromName(_ id: String) -> RefV {
+        switch id {
+        case "classes": return CLASSES
```

Comment:
It's not required but, at every other `swift` in the driver we've aligned the `return` statements in the same column.